shared/components/headerbox.js

```diff
@@ -0,0 +1,51 @@
+import React, {PropTypes, Component} from 'react';
+import {Text, StyleSheet, View, TouchableOpacity} from 'react-native';
+
+const styles = StyleSheet.create({
+  text: {
+    fontSize: 20,
+    textAlign: 'center',
+    margin: 10
+  }
+});
+
+export default class Headerbox extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSelectCategoryA = this.handleSelectCategoryA.bind(this);
+    this.handleSelectCategoryB = this.handleSelectCategoryB.bind(this);
+    this.handleSelectCategoryC = this.handleSelectCategoryC.bind(this);
+  }
+
+  handleSelectCategoryA() { this.props.update_markers('A'); }
+  handleSelectCategoryB() { this.props.update_markers('B'); }
+  handleSelectCategoryC() { this.props.update_markers('C'); }
+
+  render() {
+    return (
+      <View style={{flexDirection:'row', flex: 1}}>
+        <TouchableOpacity
+          style={{margin: 1, backgroundColor: 'green', flex: 1}}
+          onPress={this.handleSelectCategoryA}>

```

Comment:
why do you make 'handleSelectCategoryA' function?
I think at here, you can directly use 'update_markets' function.
(e.g. onPress={this.props.update_markers('A')})